mlflow/models/cli.py

```diff
@@ -155,6 +155,46 @@ def prepare_env(
     ).prepare_env(model_uri=model_uri)
 
 
+@commands.command("generate-dockerfile")
+@cli_args.MODEL_URI_BUILD_DOCKER
+@click.option(
+    "--output-directory",
+    "-d",
+    default="generate-dockerfile-output",
```

Comment:
As @dbczumar suggested, can we change the default value to `mlflow-dockerfile`?

Reply:
I'm sorry, it might've gotten lost somewhere in stash, I should've checked before resolving